lte/gateway/c/core/oai/tasks/sgw/sgw_context_manager.c

```diff
@@ -226,8 +226,8 @@ status_code_e sgw_cm_remove_bearer_context_information(
             "Failed to free imsi64 from imsi_ue_context_htbl \n");
         return temp;
       }
-      delete_spgw_ue_state(imsi64);
     }
+    delete_spgw_ue_state(imsi64);
```

Comment:
this does not look correct to me. Should not we delete UE state when where is no more bearer contexts?

Reply:
You're correct, I was trying to resolve SPGW UE staleness but after checking that is due to the other errors on zmq assertion.

Reply:
can you update the summary?